spec/PacerSpec.js

```diff
@@ -4,6 +4,7 @@ describe('The PACER module', function () {
   const noTrailingSlashUrl = 'https://ecf.canb.uscourts.gov';
   const docketReportURL = 'https://ecf.canb.uscourts.gov/cgi-bin/DktRpt.pl';
   const singleDocUrl = 'https://ecf.canb.uscourts.gov/doc1/034031424909';
+  const singleDocAudioUrl = 'https://ecf.canb.audio.uscourts.gov/doc1/034031424909';
```

Comment:
Some audio file `View Document` forms(such as those in the `paed` court) contain this url format, the domain doesn't actually resolve or get used directly it would appear, however we need to support parsing these for the extension to work properly.